Improve navbar's hamburger responsive

Shriukan33 opened the pull request:
The navbar progressively displays fewer elements instead of hiding
everything under a hamburger menu at 992px.

Because of the addition of a new link, the links are moved to a
dropdown menu "Nos projets" at 1200px, then to a hamburger at 768px.

The donation button is always present on width > 576px, and is moved
to the hamburger menu below.

Shriukan33 commented:
@JeffAbrahamson Updated with the "Nos projets" dropdown

JeffAbrahamson commented:
Looks much better!

* When it gets really small, can we drop the word "menu" so we don't crush the logo on phones?
* Could you add "la petite mobilite"?

Commit message grammar improvements:
* less -> fewer (it's countable)
* "and 992 px"?  Do you mean at?  Is it 992 and not 990?

Shriukan33 commented:
> Looks much better!
> 
>     * When it gets really small, can we drop the word "menu" so we don't crush the logo on phones?
> 
>     * Could you add "la petite mobilite"?
> 
> 
> Commit message grammar improvements:
> 
>     * less -> fewer (it's countable)
> 
>     * "and 992 px"?  Do you mean at?  Is it 992 and not 990?

Added the Petite Mobilité to links, and edited commit comment
Because the Petite mobilité takes some extra space, I moved to a dropdown earlier at 1200px

JeffAbrahamson reviewed (approved):
LGTM.
